src/app/view/meter-read/add-bulk-reads/add-bulk-reads.component.ts

```diff
@@ -0,0 +1,131 @@
+import { Component, OnInit, ViewChild } from '@angular/core';
+import { Observable } from 'rxjs';
+import { FileuploadService } from '../../../auth/services/fileupload.service';
+import { MatTableDataSource } from '@angular/material/table';
+import { MatSort } from '@angular/material/sort';
+import { MatPaginator } from '@angular/material/paginator';
+import { AdminService, OrganizationService } from '../../../auth/services';
+import { OrganizationInformation } from '../../../models';
+
+@Component({
+  selector: 'app-add-bulk-reads',
+  templateUrl: './add-bulk-reads.component.html',
+})
+export class AddBulkReadsComponent implements OnInit {
+  currentFile?: File | null;
+  progress = 0;
+  message = '';
+  pageSize: number = 10;
+  fileName = 'Please click here to select file';
+  fileInfos?: Observable<any>;
+  showmeterreadinfo: boolean = false;
+  MeterreadstatusList: any = [];
```

Comment:
remove things here which are not currently used. because this feature will not be implemented as the other ones were implemented.

We are not going to use the jobs as they are currently used. I would recommend to keep things which are currently used and don't related to the api usage